db/schema.rb

```diff
@@ -123,6 +123,14 @@
     t.index ["scheduled_at"], name: "index_good_jobs_on_scheduled_at", where: "(finished_at IS NULL)"
   end
 
+  create_table "invitations", id: :uuid, default: -> { "gen_random_uuid()" }, force: :cascade do |t|
```

Comment:
I don't think this table belongs in this commit.

Reply:
Addressed in 615e487